satellite/peer.go

```diff
@@ -222,6 +231,8 @@ func New(log *zap.Logger, full *identity.FullIdentity, db DB, config *Config) (*
 			AuditCount:            config.Node.AuditCount,
 			NewNodeAuditThreshold: config.Node.NewNodeAuditThreshold,
 			NewNodePercentage:     config.Node.NewNodePercentage,
+			//ToDo: Add Custom Version for Node Selection instead of the same as the satellite runs on
+			Version: versionInfo.Version.String(),
```

Comment:
@jtolds This is currently setting the selection version for the Nodeselection to be equal to the Satellite Version.
Do we wanna add a flag for that?

Reply:
yeah i think we want a flag that defaults to the satellite version

Reply:
The Overlay cache seems to have a default value that isnt used. Is that possible?

Reply:
i hooked up the flag to the existing config struct